storage/innobase/btr/btr0scrub.cc

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2014, Google Inc.
 // Copyright (c) 2017, MariaDB Corporation.
 
-/**************************************************//**
+/**************************************************/ /**
```

Comment:
Optional but if you have to repush due to other comments
```suggestion
/**
```